db/data_migration/20150317165803_republish_case_studies_to_render_on_government_frontend.rb

```diff
@@ -0,0 +1 @@
+DataHygiene::PublishingApiRepublisher.new(CaseStudy.where(:state => Edition::PUBLICLY_VISIBLE_STATES)).perform
```

Comment:
Might it be easier to just do two separate queries, one for the publicly_visible_stats and one for the unpublished ones?